devices/stm32mp157c-dk2

```diff
@@ -0,0 +1,18 @@
+{% set PROJECT = PROJECT|default("") %}
+{% extends PROJECT+"qemu.jinja2" %}
+
+{% set DOWNLOAD_COMMANDS = DOWNLOAD_COMMANDS|default("udhcpc; cd /tmp; wget") %}
+{% set UNPACK_COMMANDS = UNPACK_COMMANDS|default("tar -C / -xzf") %}
+{% set DEPLOY_OS = DEPLOY_OS|default("oe") %}
+{% set BOOT_OS_PROMPT = BOOT_OS_PROMPT|default("root@stm32mp1") %}
+{% set DEPLOY_TARGET = DEPLOY_TARGET|default("flasher") %}
+
+{% set boot = boot|default(false) %}
+{% set use_context = unpack_commands|default(false) %}
```

Comment:
I don't understand this. Where is the 'unpack_commands' variable coming from?

Reply:
I don't understand that either... fixing! =)